docs/source/intro/install_conda.rst

```diff
@@ -83,12 +76,31 @@ This is the recommended method for users on shared systems and systems where the
        First name: my
        Last name: name
        Institution: where-i-work
-
+       
+   .. tip:: 
+   
+        Non-default ports are used for *database cluster* and the RabbitMQ server, you can pass them using ``--db-port`` and ``--broker-port`` options respectively.
```

Comment:
Should this say (?):
```suggestion
        In case that non-default ports are used for the *database cluster* and the RabbitMQ server, you can pass them using ``--db-port`` and ``--broker-port`` options respectively.
```

Reply:
sure